### ISSUE TITLE
[Partners] FED: The local and international partner lists display differently. 

### ISSUE THREAD

**Issue by btornes** [InterActionNGO/ngoaidmap#40]
Can you format the international list like the local one: sorted and one-per-line

See https://ngoaidmap.org/projects/2068 for example of current difference

**Comment by btornes** [comment]
Hi Megan/Brian - Here is a response from Robert on this one. Seems like some larger issues need to be taken care of?

Becky,
I think there's a mix up with this feature.  Not only is/was the formatting differently, but upon viewing more projects it appears that the old functionality is still in play and conflicting with the new.  For instance, the project 2068 on staging does not have any partners in the partnerships table (these relations were added to production db), but it has the old text data, which it is now displaying.  The old fields should not be used anywhere.

OLD partner db fields:
projects.implementing_organization   (intl partners)
projects.partner_organizations   (local partners)

Code using the above references should be scrubbed.  It appears they're still in play though, based on the recent GitHub pull: https://github.com/InterActionNGO/ngoaidmap/pull/58/commits/b0b1226490b3d1cef999cf3a73f78cfc7b3d088d…; and several projects I've looked at that are mixing content from the old text fields into the display for international and local partners in the sidebar.  I've noticed, for example, when a new record in the new "partnerships" table is not yet available for a project, it's defaulting to displaying the old text field content, instead of correctly not displaying anything.   Another oddity is this duplication of int'l partners:  https://ngoaidmap.org/projects/138 ; which shows that two conflicting templates (old vs new configuration are in play).

All references to partners should utilize the new "partnerships" table.   If there's no partner relations for a project in there, nothing should display on the front end.  Based on my test project, https://ngoaidmap.org/projects/20455, it appears to be pulling from the partnerships table correctly, portioning each org based on its "international" value, but this is when there are no old text values in the fields listed above.
Regards,
Robert
